tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "lib": ["es5", "es2015.collection", "es2015.Iterable", "es2015.promise", "dom"],
     "types": [],
     "module": "esnext",
-    "moduleResolution": "node",
+    "moduleResolution": "nodenext",
```

Comment:
This is tricky. Changing this makes the code incompatible with Webpack 4, react-scripts and other legacy tools.

Try dry-run to live and see how many packages fail

Reply:
Alright, will do 👍